src/main/java/lotto/Input.java

```diff
@@ -0,0 +1,73 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Scanner;
+
+public class Input {
+    Scanner scanner = new Scanner(System.in);
+
+    public String inputLine(String guide) {
+        System.out.println(guide);
+        return scanner.nextLine();
+    }
+
+    public int inputInt(String guide) {
+        System.out.println(guide);
+        return scanner.nextInt();
+    }
+
+    public List<Numbers> inputWinningNumber() {
+        String strWinningNumber = scanner.nextLine();
+        String[] stringNumber = strWinningNumber.split(",");
+        Numbers[] lottoNum = Numbers.values();
+        List<Numbers> sixNumbers = new ArrayList<>();
+
+        for (String num : stringNumber) {
+            sixNumbers.add(lottoNum[Integer.parseInt(num)]);
+        }
```

Comment:
```suggestion
        List<Numbers> lottoNumbers = Arrays.stream(stringNumber)
            .map(number -> Integer.parseInt(number))
            .map(number -> Numbers.get(number))
            .collect(Collections.toList());
```
스트림을 활용하면 여러 변수 선언을 줄일 수 있겠죠? 대신 `Numbers` 에서 메소드가 하나 필요하네요.
스트림을 사용해보는건 어때요?

Reply:
`mapToInt`을 활용하는 방법도 있겠군요.